test/integration/instrumentation/http-outbound.test.js

```diff
@@ -159,42 +160,15 @@ test('external requests', async function (t) {
     })
   })
 
-  await t.test('should not duplicate the external segment', function (t, end) {
```

Comment:
if i nock this, it will not create the other child segments. frankly not sure the point of this tests so i just removed